dev_guide/projected_volumes.adoc

```diff
@@ -0,0 +1,368 @@
+[[dev-guide-projected-volumes]]
+= Projected Volumes
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+A _projected volume_ maps several existing xref:volumes.adoc#dev-guide-volumes[volume sources] into the same directory.
+
+Currently, the following types of volume sources can be projected:
+
+* xref:secrets.adoc#dev-guide-secrets[Secrets]
+* xref:configmaps.adoc#dev-guide-configmaps[Config Maps]
+* xref:downward_api.adoc#dev-guide-downward-api[Downward API]
+
+[NOTE]
+====
+All sources are required to be in the same namespace as the pod.
+====
+
+Projected volumes allow the user to map any combination of these volume sources into a single directory
+
+* automatically populate a single volume with the keys from multiple secrets, configmaps, and with downward API information,
+so that I can synthesize a single directory with various sources of information
+* populate a single volume with the keys from multiple secrets, configmaps, and with downward API information,
+explicitly specifying paths for each item, so that I can have full control over the contents of that volume
+
+
+== Example Scenarios
+// https://github.com/kubernetes/community/blob/master/contributors/design-proposals/all-in-one-volume.md
+
+The following general scenarios show how you can use projected volumes.
+
+* *ConfigMap, Secrets, Downward API.* Projected volumes allow you to to deploy containers with configuration data that includes passwords.
+An application using these resources could be deploying OpenStack on Kubernetes. The configuration data may need to be assembled differently depending on if the services are going to be used for production or for testing. If a pod is labeled with production or testing, the downward API selector metadata.labels can be used to produce the correct OpenStack configs.
+
+* *ConfigMap + Secrets.* Projected volumes allow you to deploy containers involving configuration data and passwords.
+For example, you might execute an Ansible playbook stored as a configmap, with some sensitive encrypted tasks that are decrypted using a vault password file.
+
+* *ConfigMap + Downward API.* Projected volumes allow you to generate a config including the pod name (available via the `metadata.name` selector). This application can then pass the pod name along with requests in order to easily determine the source without using IP tracking.
+
+* *Secrets + Downward API.* Projected volumes allow you to use a secret as a public key to encrypt the namespace of the pod (available via the `metadata.namespace` selector).
+This example allows the operator to use the application to deliver the namespace information securely without using an encrypted transport.
+
+## Example Pod Specifications
```

Comment:
To be consistent, I'd change the ##s above (and there's a few more in the doc) to be ==s.